Automattic/wp-calypso
# Add a beta feedback form to the Site Editor

#### Changes proposed in this Pull Request

Registers a plugin in the `wpcom-block-editor` to allow users to enter FSE beta feedback within the site editor.  This plugin then sends the users message through the editor to calypso via the iframe-bridge-server so calypso can then handle sending the zendesk ticket with the users feedback, locale, site data, etc.

![Screen Shot 2021-10-07 at 7 17 48 AM](https://user-images.githubusercontent.com/28742426/136374175-235762d6-8aba-4c47-a4cc-bfe1e5604f97.png)

#### Testing instructions



* Checkout this PR
* Build and sync the `wpcom-block-editor` to your sandbox: `yarn dev --sync` from `apps/wpcom-block-editor`
* Build local calypso
* Load the Site Editor on an FSE enabled site in local calypso.
* Find the comment icon in the top right of the editor.  Click it to open the feedback form.
* You can enter feedback and 'send'.  However, at this point only cylons have the zendesk access required to check the message. 
* We will have to verify your Zendesk ticket was sent after access is granted.
* Load the page/post editor and smoke test.



Related to #
